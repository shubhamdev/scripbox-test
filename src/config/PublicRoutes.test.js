import React from "react";
import { unmountComponentAtNode } from "react-dom";
import { shallow } from "enzyme";
import PublicRoutes from "./PublicRoutes";

let container = null;

beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe("PublicRoutes", () => {
  it('should render PublicRoutes correctly in "debug" mode', () => {
    const component = shallow(<PublicRoutes debug />);
    expect(component).toMatchSnapshot();
  });
  it("should render PublicRoutes correctly with no props", () => {
    const component = shallow(<PublicRoutes />);
    expect(component).toMatchSnapshot();
  });
});
