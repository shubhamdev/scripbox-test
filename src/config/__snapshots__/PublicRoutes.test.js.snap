// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PublicRoutes should render PublicRoutes correctly in "debug" mode 1`] = `
<Route
  debug={true}
  render={[Function]}
/>
`;

exports[`PublicRoutes should render PublicRoutes correctly with no props 1`] = `
<Route
  render={[Function]}
/>
`;
